Sources/TensorFlow/Core/BroadcastPullback.swift

```diff
@@ -0,0 +1,62 @@
+// Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/// Pullback for broadcasting.
+public struct BroadcastPullback {
```

Comment:
I think "broadcasting pullback" is a slightly clearer name: "a pullback that performs broadcasting".
This sounds clearer than "a *broadcast* pullback" - using "broadcast" like a participle sounds less natural to me.
```suggestion
public struct BroadcastingPullback {
```